README.md

```diff
@@ -345,6 +346,16 @@ To enable deployment of the monitoring module, you need to enable the following
 deploy_monitoring = true
 ```
 
+**ASG_WAIT**
+
+In order to be able successfully to attach existing volume to new instance if the instance is replaced there is a function in the 01_wait_node_count.sh.tpl file that will wait
+all instances to be terminated in order to continue attaching the existing volume. This can take a lot of time if you want to test something. That's why we introduced ability
```

Comment:
this is misleading, change to: "that will wait for the termination process to finish to continue attaching the existing volume. This is a considerably slow, but necessary operation. For testing purposes this operation can be skipped by configuring the following variable".